OrbitGl/ThreadTrack.cpp

```diff
@@ -90,12 +90,12 @@ bool ThreadTrack::IsTimerActive(const TimerInfo& timer_info) const {
     return std::optional<Color>{};
   }
 
-  // See Orbit.h for more information about the manual instrumentation API.
-  constexpr int kColorArgumentIndex = 2;
-  constexpr uint64_t kColorAuto = 0x00000001;
-  CHECK(timer_info.registers_size() > kColorArgumentIndex);
-  uint64_t color_arg = timer_info.registers(kColorArgumentIndex);
-  return color_arg != kColorAuto ? ToColor(color_arg) : std::optional<Color>{};
+  orbit_api::Event event = ManualInstrumentationManager::ApiEventFromTimerInfo(timer_info);
+  if (event.color == orbit::Color::kAuto) {
+    return std::optional<Color>{};
```

Comment:
I guess you can return `std::nullopt` to avoid specifying to full type.

Reply:
Done, thanks!